zhaquirks/sonoff/snzb03p.py

```diff
@@ -0,0 +1,64 @@
+"""Sonoff SNZB-03P - Zigbee motion sensor."""
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import EntityPlatform, EntityType, NumberDeviceClass, QuirkBuilder
+from zigpy.quirks.v2.homeassistant import UnitOfTime
+import zigpy.types as t
+from zigpy.zcl.clusters.measurement import OccupancySensing
+from zigpy.zcl.clusters.security import IasZone
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class LastIlluminationState(t.enum8):
+    """Last measureed state of illumination enum."""
+
+    Dark = 0x00
+    Light = 0x01
+
+
+class SonoffIlluminationCluster(CustomCluster):
+    """Custom Sonoff illumination cluster."""
+
+    cluster_id = 0xFC11
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        last_illumination_state = ZCLAttributeDef(
+            id=0x2001,
+            type=LastIlluminationState,
+            access="rp",
+            is_manufacturer_specific=True,
+        )
+
+    @property
+    def _is_manuf_specific(self):
+        return False
```

Comment:
I think we can use `manufacturer_id_override: t.uint16_t = foundation.ZCLHeader.NO_MANUFACTURER_ID` instead. See this PR as an example: https://github.com/zigpy/zha-device-handlers/pull/3910/files#diff-6c9212ca1af0b4604a20edb89247ceb6d30b010041e81fde812465a2f73a3ccbR24

Reply:
Can not be used.

Replace with `manufacturer_id_override: t.uint16_t = foundation.ZCLHeader.NO_MANUFACTURER_ID` is broken:

- eWeLink SNZB-03P is always detected
- Last illumination state is Unavailable

![2025-03-26 10 11 35](https://github.com/user-attachments/assets/cff36a2b-cf6c-4e52-b4b0-49396b71984f)


With `_is_manuf_specific(self): return False` is right:

![2025-03-26 10 14 24](https://github.com/user-attachments/assets/be4a4c50-bccd-49d4-9c99-f4fc1a93d149)

Reply:
> eWeLink SNZB-03P is always detected
> Last illumination state is Unavailable

Adding `manufacturer_id_override` on the `SonoffIlluminationCluster` won't influence the motion entity/attribute, since it's on another cluster.

And last illumination sensor being unavailable also can't be a result of adding that. Did you see any errors in the logs when adding `manufacturer_id_override` to that cluster, similar to how it's done in the linked PR above?

Maybe you forgot to import `from zigpy.zcl import foundation` at the top?